src/services/sanity.js

```diff
@@ -489,6 +489,9 @@ export function getSortOrder(sort= '-published_on', groupBy)
         case "slug":
             sortOrder = groupBy ? 'name' : "title";
             break;
+        case "name":
+            sortOrder = "name";
+            break;
         case "popularity":
             sortOrder = "popularity";
             break;
```

Comment:
This is a smidge cleaner.
```suggestion
        case "name":
        case "popularity":
            sortOrder = sort;
            break;
```